lang/java/reef-common/src/main/java/org/apache/reef/util/OSUtils.java

```diff
@@ -77,6 +77,7 @@ public static long getPID() {
         final Process process = new ProcessBuilder()
             .command("bash", "-c", "echo $PPID")
             .start();
+        process.waitFor();
```

Comment:
We should probably introspect on the return value and react to cases where it isn't `0`, right?

Reply:
I was thinking non 0 exit will trigger parsing error and exception would be caught.
But for clearer logic, I will update that.